content/S2016/S16-event-code-party.md

```diff
@@ -0,0 +1,22 @@
+title: Code Party 
+Date: 2016-07-14
```

Comment:
oh can you actually update this to be today's date? it's putting the event before the event today because you made your PR earlier